libs/designsystem/src/lib/components/radio/radio.component.scss

```
@@ -6,26 +6,32 @@ $radio-icon-mark-size: 60%;
 $spacing-to-edge: map-get($checkbox-radio-spacing, 'to-edge');
 $spacing-to-label: map-get($checkbox-radio-spacing, 'to-label');
 
+@mixin wrapper-top-bottom-padding($target-height) {
+  $padding: calc((#{$target-height} - #{$radio-icon-size}) / 2);
+
+  padding-top: $padding;
+  padding-bottom: $padding;
+}
+
 :host {
   display: inline-block;
-
   &.has-label {
     display: block;
 
     div.wrapper {
       position: relative;
       display: inline-flex;
-      align-items: center;
+      align-items: flex-start;
       vertical-align: top;
       padding-right: $spacing-to-label; // Ensure same padding on right side of label
-      min-height: map-get($checkbox-radio-sizes, 'md');
+      @include wrapper-top-bottom-padding(map-get($checkbox-radio-sizes, 'md'));
```

Consider inlining mixin for readability (and move above `padding-right`):
```suggestion
      padding: (map-get($checkbox-radio-sizes, 'md') - $checkbox-icon-size) / 2;
```

---

Resolved by: 4e49a6f